specification/azurefleet/AzureFleet.Management/common.tsp

```diff
@@ -1,19 +1,44 @@
 import "@typespec/http";
 import "@typespec/rest";
 import "@typespec/versioning";
-import "@azure-tools/typespec-autorest";
 import "@azure-tools/typespec-azure-core";
 import "@azure-tools/typespec-azure-resource-manager";
 
 using TypeSpec.Http;
 using TypeSpec.Rest;
 using TypeSpec.Versioning;
-using Autorest;
 using Azure.ResourceManager;
 using OpenAPI;
 
 namespace Microsoft.AzureFleet;
 
 @doc("ApiError for Fleet")
-@useRef("../../../../../compute/resource-manager/Microsoft.Compute/common-types/v1/common.json#/definitions/ApiError")
-model ApiError {}
+model ApiError is ApiErrorBase {
```

Comment:
There is error type `ErrorDetail` in common-types, please use that:
```typescript
model ApiError {
  ...ErrorDetail,
  innererror?: InnerError,
}

@doc("Inner error details.")
model InnerError {
  @doc("The exception type.")
  exceptiontype?: string;

  @doc("The internal error message or exception dump.")
  errordetail?: string;
}
```

Reply:
Our Api does not use this errorDetail. It uses what [Microsoft.Compute](https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=/src/Shared/Lib/ResourceProvider.Service/API/RpcEntityModel.cs&version=GBmaster&line=123&lineEnd=125&lineStartColumn=1&lineEndColumn=43&lineStyle=plain&_a=contents) uses. Reference in [Fleet](https://msazure.visualstudio.com/One/_git/Compute-CPlat-Fleet?path=/src/Dev/FleetManager.ApiEntityModel/ApiEntityModel-FleetVMScaleSets.cs&version=GCec3796d65d47fc5b0e4170ea49fc2a4d51e768e9&line=51&lineEnd=51&lineStartColumn=1&lineEndColumn=31&lineStyle=plain&_a=contents)